daemon/container_operations_unix.go

```diff
@@ -607,10 +607,40 @@ func (daemon *Daemon) getNetworkSandbox(container *container.Container) libnetwo
 	return sb
 }
 
+func (daemon *Daemon) updateNetworks(container *container.Container, idOrName string, updateSettings bool) (libnetwork.Network, error) {
```

Comment:
minor nit. The appropriate function name would be `updateNetworkConfig` since it does update both the container.Config and also the network specific settings. Can you please change that ?